app/views/admin/loan_questions/_loan_types.slim

```diff
@@ -0,0 +1,25 @@
+= f.input "loan_types" do
+  - if (@loan_question.parent || @parent_id)
+    = f.input_field :override_associations, collection: CustomField::OVERRIDE_ASSOCIATIONS_OPTIONS,
+      class: 'form-control override-parent'
```

Comment:
Yes. Remember the funky looking select box earlier? That was this. It needs form-control to be styled.